README.md

```diff
@@ -771,28 +777,85 @@ let numbers = create([Number], [1, 2, 3, 4]);
 </ul>;
 ```
 
-# Observable Microstates
+# Streaming State
 
-By themselves microstates are purely functional. They have no builtin concept of identity,
-time or sequencing. However, it is precisely because of these properties that they can be
-used with almost any state management solution that does. In fact, Microstates comes bundled
-out of the box with the ability to convert any microstate into a stream of transitions using
-the the Observable API.
+A microstate represents a single immutable value and provides transitions to derive the next value. Microstates may be considered pure function of type and value. In addition, Microstate transitions are pure, meaning they have no side-effects or, in other words, they do not affect anything outside of themselves. These are powerful constraints that provide us with guarantees that we can leverage to build efficient reactive systems.
 
-Microstates provides an easy way to convert a Microstate which represents a single value into
-a Observable stream of values. This is done by passing a Microstate to `Observable.from` function.
-This function will return a Observable object with a `subscribe` method. You can subscribe to the
-stream by passing an observer to the subscribe function. Once you subscribe, you will synchronously
-receive a microstate with middleware installed that will cause the result of transitions to be pushed
-through the stream.
+To bring a reactive system to life, we need to capture the latest state in a sequence of states. In component driven reactive engines like React, Vue, Ember & Angular the state needs to emerge at the root of the component tree. When a new state is emitted, we need to apply the state to the root component and allow the state to cascade down the component tree.
 
-You should be able to use to any implementation of Observables that supports `Observer.from` using [symbol-observable](https://github.com/benlesh/symbol-observable). We'll use `RxJS` for our example.
+Microstates has a mechanism to working with Microtates in reactive environments. It is called Identity. When you create an Identity from a Microstate, you get an object that has the same shape the original microstate. Every composed microstate becomes an identity and every transition gets wrapped in side effect emitting behaviour specific to that identity constructor. Let’s look at the features of the Identity and how they benefit reactive environments.
```

Comment:
```suggestion
Microstates has a mechanism for working with reactive environments: Identity. When you create an Identity from a Microstate, you get an object that has the same shape the original microstate. Let’s look at the features of the Identity and how they benefit reactive environments.
```

I think there was a little too much context given up front there. Better to cut it down a bit and let the example explain how it's used.